rpcs3/Emu/RSX/Common/texture_cache.h

```diff
@@ -205,7 +205,7 @@ namespace rsx
 
 		struct sampled_image_descriptor : public sampled_image_descriptor_base
 		{
-			image_view_type image_handle = 0;
```

Comment:
May be non-pointer

Reply:
`image_view_type image_handle = {};`
this is portable between integrals and pointers.